test/CpmsCommonTest/Utility/LoggerAwareTraitTest.php

```diff
@@ -3,13 +3,14 @@
 namespace CpmsCommonTest\Utility;
 
 use CpmsCommonTest\Bootstrap;
+use Laminas\ServiceManager\ServiceManager;
 
 class LoggerAwareTraitTest extends \PHPUnit\Framework\TestCase
 {
-
+    /** @phpstan-ignore missingType.property */
```

Comment:
Should we remove it and add LoggerInterface type? unless i am wrong and we cannot find out about the exact type

Reply:
It just freaks out because (I assume) the trait isn't designed to be used on it's own and the`getMockForTrait` call lower down just returns a MockObject with no indication that the trait is attached to it.

Reply:
I've added a new interface and a new mock for the trait, which has fixed it.